projects/dummy_PETR/mmdet3d_plugin/models/builder.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import warnings
+
+from mmdet3d.registry import MODELS
+
+BACKBONES = MODELS
+NECKS = MODELS
+ROI_EXTRACTORS = MODELS
+SHARED_HEADS = MODELS
+HEADS = MODELS
+LOSSES = MODELS
+DETECTORS = MODELS
+SEGMENTORS = MODELS
+VOXEL_ENCODERS = MODELS
+MIDDLE_ENCODERS = MODELS
+FUSION_LAYERS = MODELS
+SEGMENTORS = MODELS
+
+
+def build_backbone(cfg):
```

Comment:
This file is not necessary?

Reply:
Those kinds of files will be delete when I move the code from project/ to mmdet3d/